apps/sms/services/js/messaging/messaging_service.js

```diff
@@ -61,7 +63,11 @@ var MessagingService = {
 
     this.initService();
 
-    this.mozMobileMessageClient = bridge.client(MOBILE_MESSAGE_CLIENT_NAME);
+    this.mozMobileMessageClient = bridge.client({
+      service: MOBILE_MESSAGE_CLIENT_NAME,
+      endpoint: new BroadcastChannel(MOBILE_MESSAGE_CLIENT_NAME + '-channel'),
```

Comment:
__question:__ Not sure if we need leave comment\todo here that it won't work for the second app instance in case it has been run when _first_ instance was alive, but then was killed/closed?